articles/app-service/overview-managed-identity.md

```diff
@@ -40,7 +40,7 @@ To set up a managed identity in the portal, you will first create an application
 
 
 > [!NOTE] 
-> To find the managed identity for your web or slot app in the Azure portal, go to the User settings section under Enterprise applications.
+> To find the managed identity for your web or slot app in the Azure portal, go to the User settings section under Enterprise applications. Usually the slot name would be like `<App Service Name>/slots/<slot Name>`
```

Comment:
This article is shared with Azure Functions, and as such "App Service Name" may prove confusing. What about just changing this to "app name" instead?

Reply:
@mattchenderson got it and updated as per the comment